hoover/search/es.py

```diff
@@ -43,13 +64,59 @@ def get(collection_id, doc_id):
         id=doc_id,
     )
 
-
-def search(query, fields, highlight, collections, from_, size, sort, aggs):
+def _get_indices(collections):
     from .models import Collection
     indices = ','.join(
         c.index for c in
         Collection.objects.filter(name__in=collections)
     )
+    return indices
+
+@convert_es_errors
+def batch_count(query_strings, collections, aggs=None):
+    def _serialize(object):
+        if object:
+            return json.dumps(
+                object,
+                separators=(',', ':'),
+            ).replace('\n', '')
+        return '{}'
+
+    def _build_query_lines(meta, query_string, aggs):
```

Comment:
`msearch` requires 2 lines for each request: a `meta` object with things like timeouts, and the search body itself. It seems clearer to keep an argument there instead of hard-coding double empty lines.